flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-oceanbase/src/test/java/org/apache/flink/cdc/connectors/oceanbase/sink/OceanBaseMetadataApplierTest.java

```diff
@@ -145,6 +145,30 @@ void testCreateTable() {
                         .build();
 
         assertThat(actualTable).isEqualTo(expectTable);
+
+        tableId = TableId.tableId("nonexistent' OR '1'='1", "tabl1");
+        metadataApplier.applySchemaChange(new CreateTableEvent(tableId, schema));
+        actualTable =
+                catalog.getTable(tableId.getSchemaName(), tableId.getTableName()).orElse(null);
+        assertThat(actualTable).isNotNull();
```

Comment:
What's the purpose of these piece code? check an invalid `TableId`

Reply:
They are valid database/table names, but it may be truncated to the wrong database/table names without processing SQL injection.